mbq/client/tests/compat.py

```diff
@@ -1,8 +1,4 @@
 try:
-    from unittest import mock
+    from unittest.mock import *  # noqa
```

Comment:
IMO we should just expose `mock` not all (or any) of it's members.

Reply:
Why not? `compat` is now just the same thing as `mock`

Reply:
Ah, nvm, I just saw your other comment